app/page.tsx

```
@@ -1,3 +1,180 @@
+          {/* Social Links */}
+          <div className="d-flex justify-content-start mb-4">
+            {socialLinks.map((link) => (
+              <a
+                key={link.name}
+                href={link.url}
+                target="_blank"
+                rel="noopener noreferrer"
+                className="me-3"
+                title={link.name}
+              >
+                <i className={`${link.icon} fa-2x`}></i>{" "}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)